core/chain-configs/src/genesis_config.rs

```diff
@@ -272,3 +272,102 @@ pub fn get_initial_supply(records: &[StateRecord]) -> Balance {
     }
     total_supply
 }
+
+// Note: this type cannot be placed in primitives/src/view.rs because of `RuntimeConfig` dependency issues.
+// Ideally we should create `RuntimeConfigView`, but given the deeply nested nature and the number of fields inside
+// `RuntimeConfig`, it should be its own endeavor.
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ProtocolConfigView {
+    /// Current Protocol Version
+    pub protocol_version: ProtocolVersion,
+    /// Official time of blockchain start.
+    pub genesis_time: DateTime<Utc>,
```

Comment:
We need to make sure that dates, `u64`-like types (BlockHeight) are serialized as strings due to JSON numbers limitation.

Reply:
I believe it is. For example, https://github.com/near/nearcore/blob/3ad75b5bc8e71b3316507cf0d0fce6b21d3f8ad5/neard/res/genesis_config.json#L3